api-reference/auth/desktop-sdks/web.mdx

```diff
@@ -28,20 +32,20 @@ yarn add @particle-network/auth-core-modal
 
 ## Configuration
 
-Before interacting with Particle, you'll need to import and configure your master configuration object; this'll be used to not only customize the embedded wallet modal which is optionally shown after logging in, but also will allow you to define required keys and authenticate your project.
-
-Although before we import and configure this object, you'll need to retrieve the aforementioned keys.
+The main configuration is set up by wrapping the application with a master configuration object. This object is essential for authenticating your project and customizing the embedded wallet modal, which can be optionally displayed after login.
 
-Regardless of the Particle Auth SDK you use, you'll have the opportunity to define your `projectId`, `clientKey`, and `appId` to authenticate your instance of Particle Auth. In addition to this, customizations can be applied to the resulting wallet modal.
+No matter which Particle Auth SDK you're using, you'll need to define your `projectId`, `clientKey`, and `appId` within this configuration object to properly authenticate your Particle Auth instance. Additionally, you can apply customizations to the wallet modal for a tailored user experience.
```

Comment:
`[you can apply customizations to the wallet modal](/guides/configuration/appearance/wallet)`